REFERENCE.md

```diff
@@ -48,8 +48,15 @@
 
 ### iptables
 
-Add management of iptables with default rule optimization and a failsafe
-fallback mode
+----------
+
+> It is **highly recommended** that you place this module in
+> ``firewalld`` mode if the underlying system supports it.
+>
+> You can do this by setting ``simp_options::firewall: firewalld` or
```

Comment:
I'm leery of the `firewalld` value in the `$simp_options::firewall` catalyst, but that is another ticket